src/perl/Permabit/BlockDevice/VDO/LVMVDO.pm

```diff
@@ -137,27 +137,6 @@ sub makeLVMConfigString {
   return join(" ", @values);
 }
 
-########################################################################
-# @inherit
-##
-sub setup {
-  my ($self) = assertNumArgs(1, @_);
```

Comment:
Potentially a stub for setup() should still exist for common bits like finding the vdoformat executable. I don't know how hard that would be to do.